app/configurations/config.hb.js

```diff
@@ -245,15 +246,15 @@ export default configMerger(walttiConfig, {
           en: 'Open-air bicycle parks',
           de: 'Fahrradstellplätze',
         },
-        url: '/assets/geojson/hb-layers/open-airbikepark.geojson',
+        url: GEOJSON_LAYERS_BASE_URL + 'open-airbikepark.geojson',
       },
       {
         name: {
           fi: '',
           en: 'Covered bicycle parks',
           de: 'Überdachte Fahrradstellplätze',
         },
-        url: '/assets/geojson/hb-layers/coveredbikepark.geojson',
+        url: GEOJSON_LAYERS_BASE_URL + 'coveredbikepark.geojson',
       },
```

Comment:
Please have a look in the current layer generation code and use the merged layers instead of these single feature type layers (for all merged layers)